### ISSUE TITLE
AE: Sprint 30

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#13323]
# Priorities
1. **Profile 2.0: Go/No-Go and launch to 25%**. This means we need to fix any critical bugs pre-Go/No-go.
2. Unfinished Sprint 29 tickets
3. Profile 2.0: Complete additional non-critical Profile 2.0 issues.
4. Logged-in homepage: Initial homepage iterations.
5. Logged-in homepage: Design intent collaboration checkpoint.
6. Additional discovery: Preferred facility and direct deposit for edu.
7. Profile 2.0: Stretch goal — if possible, launch to 50%.
8. Helping platform team: Pact work.

### PM
- [x] [PM] Schedule and conduct Go/No-go #13326 (week of 9.14)
- [x] [PM] Launch to 25% of users #13344
- [x] Schedule Design-Intent checkpoint meeting for LIH #13328
- [ ] [PM] Create list of staging users we need for logged-in homepage QA #8385
- [x] Matt to start maintaining the logged-in homepage project epic

### Design
- [x] Determine date forRedirect Request: Account page #10688 (Oct 1st)
- [x] [Design] Create error message for when a user's Military Branch does not appear in Military Information section #13220

### Build
- [ ] Profile does not properly handle a contact field update without any changes #12632
- [x] Appointment reminders via SMS checkbox does not handle a failure to create transaction #12751
- [x] [Bug] Profile 2.0: International addresses are being put through US address validation #13119
- [ ] [FE] Profile 2.0 bug: The profile does not properly handle a contact field update without any changes #12632
- [ ] Delete confirmation alert should scroll into view #12013
- [x] Remove `cancel` buttons when transactions are pending #12009
- [x] Excess margin on top of the address validation alert #11553

#### UAT issues

- [ ] [FE] Build error alert for when Veterans are not being reported as Veterans #13225 (blocked by design)
- [x] [FE] Create error message for when a user's Military Branch does not appear in Military Information section #13222
- [x] [FE] "Invalid date" on Military History period of service #13130

#### Other profile work
- [x] [FE] Match error messages with VA Profile codes #12923

### Design
- [x] Track the documentation from the LOA3/LOA1 use case/data point exercise in Github #12862
- [x] [Design] Connect with VAOS team to go over appointments #12896

#### Phase 1: Create 2-3 high-level concepts for logged-in homepage
- [x] Storyboarding #12861
- [x] Data points #12861
- [x] Whiteboarding #13251
- [x] Wireframing (2-3 high level concepts) #13251
- [x] Review with team and pick preferred concept #13251

### Dev 
- [x] [FE] Logged-in homepage: Basic implementation of suggested approach(es) #12881

## Direct deposit
- [x] Determine level of effort for integrating with BGS CH33 DDEFT Services #11545
- [ ] [BE] Create BGS CH33 DDEFT Service integration #13286

## Pact work

**Note: This is work for the platform team to help them get an integration testing tool set up. We don't have full details on this right now and will learn more through the sprint. No one needs to work on this until we've got full details and we've launched Profile to 25% of users**:

- [ ] VSA Auth Experience - Pact setup for [product name] #13336

## Unplanned/QA/508/Bug fixes
- [x] [Bug] Redirect logout issue #13625
- [X] [FE] Connected Apps - treat `Record not found` error as an empty data array #13446

**Comment by Samara-Strauss** [comment]
- Removing #13121 from this sprint since we are blocked by VA Profile.
- Removing #13255 from this sprint because I requested an additional review with Tressa and Cassandra before we meeting with the platform team.

**Comment by mshea0606** [comment]
- removing: Discovery - enabling 'Preferred Facility' selection in 'My VA Dashboard' / VA Profile #12875
